index.rst

```diff
@@ -1,60 +1,850 @@
+
+.. option:: --replace
+
+   Create a new `~lsst.daf.butler.CollectionType.RUN`-type collection for output datasets created by this method/subcommand., dropping `collections.current_run`.
+   It is an error to pass this option if `collections.current_run` is not set.
+
+.. option:: --continue
+
+   If `collections.current_run` is not set, remove the first entry from `collections.past_runs` (which must not be empty) and set `collections.current_run` to that.
+   Does nothing if `collections.current_run` is already set.
+
+.. option:: --restart
+
+   Drop *all* runs in `collections.past_runs` and `collections.current_run` (if it exists), and create and prep a new one to contain all outputs.
+
+.. option:: --unstore
+
+   If an output collection is dropped by this action (via :option:`--replace`, :option:`--restart`, or `Campaign.pop`), remove its dataset artifacts from the datastore only.
+   Not valid if no collections can be dropped by this operation.
+
+.. option:: --purge
+
+   If an output collection is dropped by this action (via :option:`--replace`, :option:`--restart`, or `Campaign.pop`), remove the collection and its datasets entirely from both the registry and the datastore.
+   Supersedes :option:`--unstore`.
+   Not valid if no collections can be dropped by this operation.
+
+.. option:: --skip-existing-in [INPUTS|CAMPAIGN|RUN]
+
+   Do not execute quanta for which all outputs already exist in the given collection category.
+
+   Unlike :option:`--trim-existing-in`, this does not modify the `~lsst.pipe.base.QuantumGraph`, but the argument choices have the same definition.
+
+.. option:: --rebuild
+
+   Rebuild the `~lsst.pipe.base.QuantumGraph` before running.
+   This option may be passed even when there is no graph (it is ignored).
+
+.. _opts-discrepancies:
+
+Discrepancy Resolution Options
+------------------------------
+
+Some options are used to resolve potential discrepancies between the `~lsst.pipe.base.QuantumGraph` and the pipeline and input collections from which it is typically built, when these are set out-of-order or the graph is imported.
+These include:
+
+.. option:: --use-task-configs
+
+   Do not rebuild or refresh the `~lsst.pipe.base.QuantumGraph` before running, but use the pipeline's tasks and configuration instead of those in the `~lsst.pipe.base.QuantumGraph`, matching them by label.
+
+   Note that changes `collections.inputs` since the graph was generated are also ignored when this option is used; those cannot be used unless the graph is at least refreshed.
+
+.. option:: --use-qg-configs
+
+   Do not rebuild or refresh the `~lsst.pipe.base.QuantumGraph` before running, but use as-is, ignoring the tasks in the pipeline completely.
+
+   Note that changes `collections.inputs` since the graph was generated are also ignored when this option is used; those cannot be used unless the graph is at least refreshed.
```

Comment:
```suggestion
   Note that changes to `collections.inputs` since the graph was generated are also ignored when this option is used; those cannot be used unless the graph is at least refreshed.
```